buildSrc/src/main/java/org/elasticsearch/gradle/testclusters/TestClustersPlugin.java

```diff
@@ -49,7 +52,13 @@
 
     @Override
     public void apply(Project project) {
-        project.getPlugins().apply(DistributionDownloadPlugin.class);
+        project.getRootProject().getPluginManager().apply(GlobalBuildInfoPlugin.class);
+        if (BuildParams.isInternal()) {
```

Comment:
We kicked the can of differentiating between internal and external builds further to here for now. We might want to apply similar internal / external plugin distinction later for that plugin too